contracts/connectors/loantoken/LoanTokenLogicStandard.sol

```diff
@@ -214,155 +225,6 @@ contract LoanTokenLogicStandard is AdvancedToken, StorageExtension {
         );
     }
 
-    function transfer(
-        address _to,
-        uint256 _value)
-        external
-        returns (bool)
-    {
-        return _internalTransferFrom(
-            msg.sender,
-            _to,
-            _value,
-            uint256(-1)
-        );
-    }
-
-    function transferFrom(
-        address _from,
-        address _to,
-        uint256 _value)
-        external
-        returns (bool)
-    {
-        return _internalTransferFrom(
-            _from,
-            _to,
-            _value,
-            allowed[_from][msg.sender]
-            /*IBZx(bZxContract).isLoanPool(msg.sender) ?
-                uint256(-1) :
-                allowed[_from][msg.sender]*/
-        );
-    }
-
-    function _internalTransferFrom(
-        address _from,
-        address _to,
-        uint256 _value,
-        uint256 _allowanceAmount)
-        internal
-        returns (bool)
-    {
-        if (_allowanceAmount != uint256(-1)) {
-            allowed[_from][msg.sender] = _allowanceAmount.sub(_value, "14");
-        }
-
-        require(_to != address(0), "15");
-
-        uint256 _balancesFrom = balances[_from];
-        uint256 _balancesFromNew = _balancesFrom
-            .sub(_value, "16");
-        balances[_from] = _balancesFromNew;
-
-        uint256 _balancesTo = balances[_to];
-        uint256 _balancesToNew = _balancesTo
-            .add(_value);
-        balances[_to] = _balancesToNew;
-
-        // handle checkpoint update
-        uint256 _currentPrice = tokenPrice();
-
-        _updateCheckpoints(
-            _from,
-            _balancesFrom,
-            _balancesFromNew,
-            _currentPrice
-        );
-        _updateCheckpoints(
-            _to,
-            _balancesTo,
-            _balancesToNew,
-            _currentPrice
-        );
-
-        emit Transfer(_from, _to, _value);
-        return true;
-    }
-
-    function _updateCheckpoints(
-        address _user,
-        uint256 _oldBalance,
-        uint256 _newBalance,
-        uint256 _currentPrice)
-        internal
-    {
-        bytes32 slot = keccak256(
-            abi.encodePacked(_user, iToken_ProfitSoFar)
-        );
-
-        int256 _currentProfit;
-        if (_newBalance == 0) {
-            _currentPrice = 0;
-        } else if (_oldBalance != 0) {
-            _currentProfit = _profitOf(
-                slot,
-                _oldBalance,
-                _currentPrice,
-                checkpointPrices_[_user]
-            );
-        }
-
-        assembly {
-            sstore(slot, _currentProfit)
-        }
-
-        checkpointPrices_[_user] = _currentPrice;
-    }
-
-    /* Public View functions */
-
-    function profitOf(
```

Comment:
This is a breaking change on the UI. Please relay this to Kevin and also update the HelperImpl to reflect this being removed. the profitOf function that is found in HelperImpl.sol is unusable with this removed.

Reply:
its already developed on TheGraph side. and I think Kevin already uses it